test/kpod_diff.bats

```diff
@@ -2,20 +2,16 @@
 
 load helpers
 
-IMAGE="alpine:latest"
+function setup() {
+    copy_images
+}
 
 function teardown() {
     cleanup_test
 }
 
 @test "test diff of image and parent" {
-    run ${KPOD_BINARY} $KPOD_OPTIONS pull $IMAGE
-    echo "$output"
-    [ "$status" -eq 0 ]
-    run ${KPOD_BINARY} $KPOD_OPTIONS diff $IMAGE
-    echo "$output"
-    [ "$status" -eq 0 ]
-    run ${KPOD_BINARY} $KPOD_OPTIONS rmi $IMAGE
+    run ${KPOD_BINARY} $KPOD_OPTIONS diff $BB
```

Comment:
Was this supposed to switch from using the alpine image to the busybox image?

Reply:
yeah its ok i think